inc/toolbox.class.php

```diff
@@ -3461,7 +3461,7 @@ public static function isAPIDeprecated(string $class): bool {
     */
    public static function isValidWebUrl($url): bool {
       return (preg_match(
-         '#^http[s]?://[a-z0-9\-_]+(\.([a-z0-9\-]+\.)?[a-z]+)?(:\d+)?(/.*)?$#i',
+         '#^http[s]?://[a-z0-9\-_]+(\.([a-z0-9\-\.]+\.)?[a-z]+)?(:\d+)?(/.*)?$#i',
```

Comment:
I think you may have many subdomains levels.
```suggestion
         '#^http[s]?://[a-z0-9\-_]+(\.([a-z0-9\-]+\.)*[a-z]+)?(:\d+)?(/.*)?$#i',
```